samples/nrf9160/serial_lte_modem/prj.conf

```diff
@@ -9,11 +9,11 @@ CONFIG_LOG_DEFAULT_LEVEL=3
 CONFIG_STACK_SENTINEL=y
 
 # Segger RTT
-CONFIG_USE_SEGGER_RTT=y
-CONFIG_RTT_CONSOLE=y
-CONFIG_UART_CONSOLE=n
-CONFIG_LOG_BACKEND_RTT=y
-CONFIG_LOG_BACKEND_UART=n
+CONFIG_USE_SEGGER_RTT=n
+CONFIG_RTT_CONSOLE=n
+CONFIG_UART_CONSOLE=y
```

Comment:
please not that using logger/console with asynchronous api will eventually lead to crash. There is a fix for that in zephyr PR but it will be merged after 2.2 release. https://github.com/zephyrproject-rtos/zephyr/pull/22355 so i would not use switch to that mode for now.